project-reports/2024/2024-Q3-Hyperledger-Aries.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Aries
+parent: 2024
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+nav_exclude: true
+---
+
+## Project Health
+
+Work on [Hyperledger Aries] continues at a healthy pace, with lots of activity
+in all of the Aries and the related-OWF sub-communities. [Aries Cloud Agent
+Python] continues to move to its 1.0.0 release, and the community around
+[Aries-VCX] is again growing, with lots of work happening in moving the
+implementation forward. [Open Wallet Foundation]-hosted [Credo-TS] and [Bifold
+Wallet] continue to evolve rapidly. [Animo Solutions], key maintainers of
+[Credo-TS], were selected by the German Government as funded participate in
```

Comment:
something not quite right with the grammar here.

Reply:
Reworded.